pom.xml

```diff
@@ -71,7 +71,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.dataformat</groupId>
             <artifactId>jackson-dataformat-yaml</artifactId>
-            <version>2.13.3</version>
+            <version>2.14.0-rc1</version>
```

Comment:
jackson-dataformat-yaml version 2.14.0-rc1 is not a production release but rather a preview. I'm not sure we want to use it in our production releases. Unfortunately the latest release (2.13.4) still includes snakeyaml 1.31 but we require snakeyaml 1.32 to resolve the current CVEs.

There are some known breaking changes in snakeyaml 1.30 -> 1.31 but given that jackson-dataformat-yaml 2.13.4 is already on snakeyaml 1.31 it may be possible to override the version of snakeyaml to 1.32/1.33 if we also upgrade jackson to 2.13.4. This would have to rolled back one jackson 2.14.0 is officially released though.